libs/experimental/langchain_experimental/data_anonymizer/presidio.py

```diff
@@ -103,18 +124,28 @@ def add_operators(self, operators: Dict[str, OperatorConfig]) -> None:
 
 
 class PresidioAnonymizer(PresidioAnonymizerBase):
-    def _anonymize(self, text: str) -> str:
+    def _anonymize(self, text: str, language: Optional[str]) -> str:
         """Anonymize text.
         Each PII entity is replaced with a fake value.
         Each time fake values will be different, as they are generated randomly.
 
         Args:
             text: text to anonymize
         """
+        if language is None:
+            language = self.supported_languages[0]
```

Comment:
this is not intuitive in my opinion, I would rather dismiss `None` value for `language` argument